### PR TITLE
Rod of the Umbral Crevice

### PR DESCRIPTION
**Pull request by phantamanta44**
Fixed all the issues mentioned in the previous PR.

(Except I'm unsure of what "EntityThrowableFake" is)

### CODE REVIEW

File: src/main/java/vazkii/botania/common/entity/EntityShadowBeam.java [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+		motionY = y;
+		motionZ = z;
+	}
+	
+	@Override
+	public void onUpdate() {
+		superUpdate();
+		
+		if(ticksExisted >= effectThreshhold)
+			Botania.proxy.sparkleFX(worldObj, posX, posY, posZ, 0.3F, 0.09F, 1F, 4F, 1, true);
```
Comment (Vazkii) [inline src/main/java/vazkii/botania/common/entity/EntityShadowBeam.java]: Have you tested this in multiplayer?

File: src/main/java/vazkii/botania/common/item/lens/ItemLens.java [hunk 2]
```diff
@@ -319,6 +319,6 @@ public ItemStack setCompositeLens(ItemStack sourceLens, ItemStack compositeLens)
 
 	@Override
 	public boolean shouldPull(ItemStack stack) {
-		return stack.getItemDamage() != STORM;
+		return stack.getItemDamage() < 5000;
```
Comment (Vazkii) [inline src/main/java/vazkii/botania/common/item/lens/ItemLens.java]: ?

File: src/main/java/vazkii/botania/common/entity/EntityShadowBeam.java (hunk 1 above)
Comment (phantamanta44) [inline src/main/java/vazkii/botania/common/entity/EntityShadowBeam.java]: Not yet. Will do.

File: src/main/java/vazkii/botania/common/item/lens/ItemLens.java (hunk 2 above)
Comment (phantamanta44) [inline src/main/java/vazkii/botania/common/item/lens/ItemLens.java]: I was originally going to use a beam of mana bursts with a lens of damage 5001, but that caused a few issues that I didn't want to deal with.

I figured that line of code might be useful if you might ever want to implement specialized lens again.

**Comment by phantamanta44** [comment]
Fixed particles in multiplayer, and changes damage source to indirect magic.

**Comment by phantamanta44** [comment]
Tell me if/when this should be merged so I can rebase with the newest content.

**Comment by TheWhiteWolves** [comment]
You're probably best rebasing it asap so the new changes can be reviewed

**Pull request closed without merging**